Vagrantfile

```diff
@@ -1,6 +1,22 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+require 'yaml'
+
+config_file = 'config.yaml'
+config = YAML.load_file("#{config_file}")
+
+# get parameters from config file
+magento_location_host = config['magento']['location']['host']
+magento_location_remote = config['magento']['location']['remote']
+magento_dir_name = config['magento']['dir_name']
+hostname = config['env']['hostname']
+memory = config['env']['memory']
+ip = config['env']['ip']
+
+magento_dir_host = magento_location_host + '/' + magento_dir_name
```

Comment:
How about the following change?
magento_location_host => projects_dir_host (the same for guest)